scripts/west_commands/runners/nrf_common.py

```diff
@@ -342,26 +342,28 @@ def program_hex(self):
                 mpi_hex_dir = Path(os.path.join(self.cfg.build_dir, 'zephyr'))
 
                 # Handle Manifest Provisioning Information
-                if self.build_conf.getboolean('CONFIG_SUIT_MPI_GENERATE'):
+                if self.sysbuild_conf.getboolean('SB_CONFIG_SUIT_MPI_GENERATE'):
                     app_mpi_hex_file = os.fspath(
-                        mpi_hex_dir / self.build_conf.get('CONFIG_SUIT_MPI_APP_AREA_PATH'))
+                        mpi_hex_dir / self.sysbuild_conf.get('SB_CONFIG_SUIT_MPI_APP_AREA_PATH'))
```

Comment:
bit confused, this Kconfig does not exist anywhere in tree?

Reply:
Approving since there is no way to extend flash runners out of tree at present but a system should be introduced in future to allow for that (which gives many additional benefits)